friends/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import ListFriends from './ListFriends';
+import Add from './addFriend';
+import './App.css';
+
+class App extends Component {
+
+  state = {
+    friends : [],
+  }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(response => {
+        console.log(response.data);
+        this.setState({friends: response.data})
+      })
+      .catch(err => console.log(err))
+  }
+
+  submitBtn = (input) => {
+    axios
+      .post('http://localhost:5000/friends', {...input})
```

Comment:
Since input is coming in as an object, you shouldn't have to spread it here. You should be able to 
```suggestion
      .post('http://localhost:5000/friends', input)
```